.github/actions/frontend-shared-version/index.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const core = require('@actions/core');
+const util = require('util');
+
+const exec = util.promisify(require('child_process').exec);
+
+/**
+ * Returns true is there is a non-zero length diff between the last published version
+ * and master within the frontend-shared/ folder.
+ */
+async function haveChangesOccurred() {
+  const parentPackage = core.getInput('parent-package');
+  const versionResult = await exec(`npm view ${parentPackage} version`);
+  const version = versionResult.stdout.trim();
+  const diffResult = await exec(`git diff v${version}..master frontend-shared/`)
```

Comment:
This ends up looking like 

`git diff v1.647.0..master frontend-shared/`

for example.

You can see a non-zero diff if you look back 1 more version

`git diff v1.646.0..master frontend-shared/`

Reply:
You could potentially simplify the output here using `--stat` (`git diff --stat ...`) to make the output in log files more readable.

Reply:
will do